kiva/fonttools/font.py

```diff
@@ -86,11 +86,6 @@ def __init__(self, face_name="", size=12, family=SWISS, weight=NORMAL,
             (type(underline) != int) or (not isinstance(face_name, six.string_types)) or \
             (type(encoding) != type(DEFAULT)):
                 raise RuntimeError("Bad value in Font() constructor.")
-        ### HACK:  C++ stuff expects a string (not unicode) for the face_name, so fix
-        ###        if needed.
-        ### Only for python < 3
-        if six.PY2 and isinstance(face_name, six.text_type):
-            face_name = face_name.encode("latin1")
```

Comment:
Do we have test coverage for this code?

Reply:
I ran the test suite under coverage, and the if statement was previously hit by the tests.  But since we only test on python 3.6 now, `six.PY2` is never true.  The `__init__` method is called in tests though yes.

Reply:
👍